pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -105,6 +106,7 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	if err != nil {
 		return err
 	}
+	analytics.TriggerAnalytics()
```

Comment:
Don't call the analytic from here, Add it in file `https://github.com/litmuschaos/chaos-operator/blob/master/cmd/manager/main.go`

Reply:
Done @chandankumar4 can you check if i have inserted it correctly.